tests/filecheck/dialects/riscv/riscv_ops.mlir

```diff
@@ -178,8 +178,8 @@
     // CHECK-NEXT: riscv.ecall : () -> ()
     riscv.ebreak : () -> ()
     // CHECK-NEXT: riscv.ebreak : () -> ()
-    riscv.directive {"directive" = ".align", "value" = "2"} : () -> ()
-    // CHECK-NEXT: riscv.directive {"directive" = ".align", "value" = "2"} : () -> ()
+    riscv.directive ".align" "2" : () -> ()
+    // CHECK-NEXT: riscv.directive ".align" "2" : () -> ()
```

Comment:
Can you please add a test for a missing value? Would like to avoid a double space.

Reply:
Added a test case in newest commit. Unfortunately it seems `filecheck` is not white space sensitive.